V1.3/README.md

```diff
@@ -22,3 +22,16 @@ a letter to Creative Commons, 444 Castro Street, Suite 900, Mountain View, Calif
 Feel free to work with it.
 
 Rudi
+
+**Replacing Nitrokey Storage PCB**
+You can replace your Nitrokey Storage hardware (PCB) within a few minutes by following these steps:
+
+1. Backup all your data from the Nitrokey. Understand that the RSA keys will be replaced and all data be destroyed!
+2. Remove the lid from the USB socket of the Nitrokey.
+3. Use a knife to open the casing until you have the PCB removed completely.
+4. Remove the smart card and the MicroSD card from the PCB and push them in the new replacement PCB.
+5. Place the PCB in the new bottom casing, the one with the CE and FC engravings.
+6. Add the other casing (with the Nitrokey label) on top of the bottom casing. Press both casings carefully together until it clicks and the gap is completely closed.
+7. Connect the Nitrokey to your computer. Start the Nitrokey App and select “Destroy Encrypted Data” from "Configuration" sub-menu. Note that this will destroy all existing RSA keys and encrypted data.
```

Comment:
`Destroy Encrypted Data` action will regenerate the AES key and save part of it on the smart card., which is required to make the EV work. It will not remove the keys from the smart card (as in, not execute the `factory reset` there).
If the latter is required, the Nitrokey App has to be executed with `--admin` parameter, and `Factory Reset` action has to be executed from `Menu -> Configuration -> Special Configure` menu.

Are keys removal required?

Reply:
Changing this text from `RSA` to `AES` should be sufficient.